app/controllers/users_controller.rb

```diff
@@ -23,10 +33,54 @@ def create
     end
   end
 
+  def update
+    if @user.update_attributes(user_params)
+      flash[:success] = t("users.updated")
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t("users.deleted")
+      redirect_to users_url
+    else
+      flash[:danger] = t("users.cant_delete")
+    end
+  end
+
   private
 
   def user_params
     params.require(:user).permit :name, :email, :password,
                                  :password_confirmation
   end
+
+  private
+
+  def logged_in_user
+    unless logged_in?
```

Comment:
Dùng cách viết sau để thay thế block unless ... end
```suggestion
   return if logged_in?
   store_location
   flash[:danger] = t("users.not_login")
   redirect_to login_url
```